grpc/src/test/java/com/linecorp/armeria/internal/server/grpc/GrpcDocServicePluginTest.java

```diff
@@ -103,6 +107,18 @@ void servicesTest() throws Exception {
                                                       m.name());
             });
         });
+        services.get(HttpJsonTranscodingTestServiceGrpc.SERVICE_NAME).methods().forEach(m -> {
```

Comment:
While adding to these tests is also useful, what do you think of also adding a decorator (e.g. `LoggingService`) to our integration test at `GrpcDocServiceTest` as well?

https://github.com/line/armeria/blob/f6c4debed749f77d71aec836e98bdffef6537273/grpc/src/test/java/com/linecorp/armeria/internal/server/grpc/GrpcDocServiceTest.java#L117

```
-                                       .addService(new TestService())
+                                       .addService(new TestService(),
+                                                   ImmutableList.of(LoggingService.newDecorator()))
```